constructr-coordination-consul/src/main/scala/com.tecsisa.constructr.coordination.consul/ConsulCoordination.scala

```diff
@@ -64,15 +64,15 @@ object ConsulCoordination {
     }
   }
 
-  private final case class ConsulToken(value: String) extends ModeledCustomHeader[ConsulToken] {
+  private final class ConsulToken(val value: String) extends ModeledCustomHeader[ConsulToken] {
```

Comment:
This is no longer a case class, because since Scala 2.12.2 it would provoke a `StackOverflowError` due to `ModeledCustomHeaderCompanion.apply()` method being called by mistake, instead of just calling the autogenerated `ConsulToken.apply()`. Note that `ModeledCustomHeaderCompanion.apply()` calls `parse()`, thus the infinite recursion.